content/docs/reference/databroker.mdx

```diff
@@ -121,26 +59,28 @@ See Kubernetes [Storage reference](/docs/k8s/reference#storage) for more informa
 
 ## Databroker Storage Connection String {#databroker-storage-connection-string}
 
-**Databroker Storage Connection String** sets the Postgres connection string that the Databroker service uses to connect to storage backend. The connection string may be provided directly in the configuration or read from a file.
+**Databroker Storage Connection String** tells Pomerium how to connect to an external PostgreSQL database. This connection string may be provided directly in the configuration or read from a file.
+
+This setting is **required** when the storage type is set to `postgres`.
```

Comment:
This is good clarification to show how these settings are related.

Reply:
Thanks; I wasn't sure how to fit this setting into a single "required"/"optional" column in the table, so I figured spelling it out in the main text would be good.